### PR TITLE
New resources added

### PR DESCRIPTION
**Pull request by dx7er**

### CODE REVIEW

**Comment by d3xt3r7** [comment]
Allow me to contribute in this repoitory.

**Comment by santosomar** [comment]
Regrettably, a number of resources incorporated in this pull request appear to infringe upon copyrights from various sources. Therefore, I am unable to approve this pull request at this time.

File: Red-Team-Operations.md
Comment (santosomar) [inline Red-Team-Operations.md:1]: This one is a good resource and does not infringe copyrights.

File: Free Paid Courses/Links.md
Comment (santosomar) [inline Free Paid Courses/Links.md:1]: Regrettably, a number of resources incorporated in this pull request appear to infringe upon copyrights from various sources.

**Review by santosomar — COMMENTED** [review]
Regrettably, a number of resources incorporated in this pull request appear to infringe upon copyrights from various sources.

**Review by santosomar — COMMENTED** [review]
Regrettably, a number of resources incorporated in this pull request appear to infringe upon copyrights from various sources.

**Review by santosomar — CHANGES_REQUESTED** [review]
Please remove all the links that violate copyrights.

File: Red-Team-Operations.md
Comment (dx7er) [reply Red-Team-Operations.md:1]: I have taken resources from GitHub and wanted to use it in this repository. There shouldn't be any copyright issue.

**Pull request closed without merging**